app/assets/stylesheets/mobile/header.css.scss

```diff
@@ -169,10 +163,20 @@ header {
 }
 
 /* This class is added when the user open the drawer */
-#app.draw > * {
-          transform: translateX(-80%);
-       -o-transform: translateX(-80%);
-      -ms-transform: translateX(-80%);
-     -moz-transform: translateX(-80%);
-  -webkit-transform: translateX(-80%);
+.draw {
```

Comment:
I was talking about Opera Mini, but I don't know if the two projects are still supported.